### PR TITLE
Adding fastft as counted content and negative offset time

### PR DESCRIPTION
**Pull request by akilananb**
Added fastft as counted content
Negative Offset should be zero

### CODE REVIEW

File: server/transforms/time.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = (event) => {
 		metrics.count('pipeline.transforms.time.count', 1);
 
 		// the actual time of the event
-		const offset = event.pluck('time.offset') || 0;
+		const offset = event.pluck('time.offset')>0? event.pluck('time.offset'): 0;
```
Comment (commuterjoy) [inline server/transforms/time.js]: @akilananb can you please fix your editors space settings? Having all characters joined together makes the code hard to read?

File: server/transforms/counted-content.js [hunk 2]
```diff
@@ -27,6 +30,10 @@ module.exports = (event) => {
 			isCounted = capiv1.metadata.brand.some(i => brands.indexOf(i.term.id) >= 0);
 		}	
 		
+		if(!barrier && !isCounted && url && url.indexOf('/fastft')!==-1){
```
Comment (commuterjoy) [inline server/transforms/counted-content.js]: We've actually just tried to remove all the URL sniffing from Spoor, is there a reason why we can just add the `brand` id to the list of counted brands?

File: server/transforms/time.js (hunk 1 above)
Comment (akilananb) [inline server/transforms/time.js]: ya sure i'll do that

File: server/transforms/counted-content.js (hunk 2 above)
Comment (akilananb) [inline server/transforms/counted-content.js]: As of now @markstephens mentioned to have all fastft content as counted one

Comment (markstephens) [inline server/transforms/counted-content.js]: Fast FT consumption doesn't have (or isn't sent with)  a uuid so it can't be looked up from the brands.

Comment (commuterjoy) [inline server/transforms/counted-content.js]: Yup, but doesn't https://github.com/Financial-Times/spoor-enrichment/blob/master/server/utils/counted-brands.js detect that already?

Comment (commuterjoy) [inline server/transforms/counted-content.js]: @markstephens Can the fast ft team not fix that?

Comment (markstephens) [inline server/transforms/counted-content.js]: No, there's no uuid. So no capi lookup, and therefore no brand to check.

Comment (markstephens) [inline server/transforms/counted-content.js]: Dunno, do fast ft posts have a uuid?

Comment (matthew-andrews) [inline server/transforms/counted-content.js]: yes they do and they're in the content api

**Comment by commuterjoy** [comment]
@akilananb can you pull out the negative time offset in to a separate pull request please? I'm closing this as spoor isn't the right place to fix the fast ft is counted content debate.

**Pull request closed without merging**